### PR TITLE
Added allowTrashedSlugs section

### PR DESCRIPTION
**Pull request by Adrion**
Documentation about https://github.com/octobercms/library/pull/388

### CODE REVIEW

**Comment by Adrion** [comment]
I added the section in the sluggable part since it's a Sluggable trait property.
However I also add a link inside the soft delete section.

My english may needs a little review though 😄

File: database-traits.md [hunk 1]
```diff
@@ -461,6 +471,12 @@ Under these same conditions, when the primary model is restored, all the related
     // Restore the user and comments
     $user->restore();
 
+<a name="soft-delete-with-sluggable-trait"></a>
+### Soft Delete with Sluggable trait
+
+By default Sluggable trait will ignore soft deleted models when the slug is generated.
+In order to make the model restoration much painless [checkout the Sluggable section](#sluggable-with-softdelete-trait).
```
Comment (w20k) [inline database-traits.md]: `By default` -> `By default,`
`much painless` -> `more painful`

**Comment by w20k** [comment]
@Adrion no worries, we're here to help ;) Found only two, @LukeTowers could you review ;)

File: database-traits.md [hunk 2]
```diff
@@ -102,6 +102,16 @@ Use the `slugAttributes` method to regenerate slugs when updating a model:
     $user->slugAttributes();
     $user->save();
 
+<a name="sluggable-with-softdelete-trait"></a>
+### Sluggable with SoftDelete trait
+
+By default, soft deleted models are ignored when the slug is generated.
+You might want to prevent slug duplication when recovering soft deleted models.
+
+Set the `$allowTrashedSlugs` attribute to `true` will prevent slug generation from ignoring soft deleted models.
```
Comment (LukeTowers) [inline database-traits.md]: Change to 
> `Set the `$allowTrashedSlugs` attribute to `true` in order to take into account soft deleted records when generating new slugs.

**Comment by Adrion** [comment]
@LukeTowers thanks for the review 👍

**Pull request merged**